Bump deps

redallen opened the pull request:
Close https://github.com/patternfly/patternfly-react/issues/2667 and https://github.com/patternfly/patternfly-react/issues/2674

redallen committed:
upgrade deps

redallen committed:
bump deps for typescript support

ssilvert commented:
This PR doesn't transpile.

node_modules/@patternfly/react-table/dist/js/components/Table/base/body-row.d.ts:5:44 - error TS2339: Property 'body' does not exist on type '{ table?: string | undefined; header?: { wrapper?: string | Function | Component<any, {}, any> | ComponentClass<any, any> | FunctionComponent<any> | undefined; row?: string | ... 4 more ... | undefined; cell?: string | ... 4 more ... | undefined; } | undefined; body?: { ...; } | undefined; } | undefined'.

5     renderers: RenderersTypes['renderers']['body'];
                                             ~~~~~~


Found 1 error.

redallen commented:
Make sure you `npm install` before you run it. That error I fixed upstream in https://github.com/patternfly/patternfly-react/issues/2674 and is in the versions included in this PR:
```
    "@patternfly/react-core": "^3.88.2",
    "@patternfly/react-table": "^2.17.5",
```
I bumped them again for you, could you give those a try? They work fine for me locally:
```sh
➜  systemjs-react git:(chore/bump-deps) ✗ rm -rf node_modules 
➜  systemjs-react git:(chore/bump-deps) ✗ npm i

> core-js@2.6.9 postinstall /home/redallen/src/systemjs-react/node_modules/core-js
> node scripts/postinstall || echo "ignore"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)

npm WARN systemjs-react@1.0.0 scripts['server'] should probably be scripts['start'].

added 353 packages from 273 contributors and audited 5564 packages in 6.552s
found 1 high severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
➜  systemjs-react git:(chore/bump-deps) ✗ npm run build

> systemjs-react@1.0.0 build /home/redallen/src/systemjs-react
> tsc

➜  systemjs-react git:(chore/bump-deps) ✗ echo $?
0
```

dlabrecq commented:
Deleting the package-lock.json file and re-installing resolves the build issue. However, I encountered some other run-time issues.

1. The systemjs v5.0.0 package uses `system.js`, not `system.src.js`
2. `System.config` does not appear to be supported in versions >= 2.0.0
3. May need to replace `System.config` with import maps?

I was able to get a little further using systemjs v0.20.17, but made the following additions to `systemjs.config.js`. All css and js files are loading for me, but the browser console shows an `unexpected '.' token` error.

<img width="484" alt="Screen Shot 2019-10-26 at 12 48 28 AM" src="https://user-images.githubusercontent.com/17481322/67614486-d0605600-f78b-11e9-916d-8cb60309217a.png">

<img width="636" alt="Screen Shot 2019-10-26 at 12 48 50 AM" src="https://user-images.githubusercontent.com/17481322/67614493-d9512780-f78b-11e9-8374-c981f379b9df.png">

<img width="718" alt="Screen Shot 2019-10-26 at 12 49 07 AM" src="https://user-images.githubusercontent.com/17481322/67614495-deae7200-f78b-11e9-8c16-3d8e1e97e060.png">
